### DISCUSSION TITLE
How do I update our Intergroup contact information?

### DISCUSSION THREAD

**Discussion by stefanklehmann** [code4recovery/12-step-meeting-list#890] [Q&A] [answered]
I've looked through the plugin settings and can't seem to figure out how to update the contact information that appears on the meeting guide mobile app.  the website is https://aavictoria.ca and our phone number appear on the app as 250-980-3826, but it should be 250-383-0415.  Can anyone tel me where I update this information?

**Comment by Grateful-Viv** [comment, marked as answer]
You should contact GSO for this. You can reach them by email appsupport@aa.org
